packages/eslint-plugin/README.md

```diff
@@ -97,6 +97,41 @@ The rule requires an options object containing:
 
 Ban references to the `React` global namespace (in favor of explicitly importing React). Implicit global references cause problems for API Extractor and potentially other tools.
 
+### `no-restricted-imports`
+
+Prevents imports from `forbidden` packages. If a corresponding `preferred` import is provided, the lint error will be automatically fixable.
+
+**Example Configuration:**
+
+```
+"@fluentui/no-restricted-imports": [
+  'error',
+  {
+    paths: [
+      {
+        forbidden: ['@fluentui/react-theme', '@griffel/react`],
+        preferred: '@fluentui/react-components',
+      },
```

Comment:
```suggestion
      {
        "@fluentui/react-theme": "@fluentui/react-components",
        "@griffel/react": "@fluentui/react-components",
      },
```
```suggestion
      {
        forbidden: ['@fluentui/react-theme', '@griffel/react`],
        preferred: '@fluentui/react-components',
      },
       {
        forbidden: ['@fluentui/react-theme', '@griffel/react`],
        preferred: '@fluentui/react-components',
      },
```

Not blocking the PR as the rule does what our requirements for it are, but here are ideas to make it more extensible